internal/commands/push/command.go

```diff
@@ -195,10 +195,38 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 
 	if cmd.inputs.IncludeDependencies {
-		if err := clients.Realm.ImportDependencies(appRemote.GroupID, appRemote.AppID, uploadPathDependencies); err != nil {
+		installDependencies := func() error {
+			s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+			s.Suffix = " Installing dependencies: starting..."
+
+			s.Start()
+			defer s.Stop()
+
+			if err := clients.Realm.ImportDependencies(appRemote.GroupID, appRemote.AppID, uploadPathDependencies); err != nil {
+				return err
+			}
+
+			status := realm.DependenciesStatus{State: realm.DependenciesStateCreated}
+			for status.State == realm.DependenciesStateCreated {
+				var err error
+				status, err = clients.Realm.DependenciesStatus(appRemote.GroupID, appRemote.AppID)
+				if err != nil {
+					return err
+				}
+				s.Suffix = fmt.Sprintf(" Installing dependencies: %s...", status.Message)
```

Comment:
[question] maybe you can try to test this out (either by running the command and watching closely or mocking the realm client to suit your needs), but do we experience any "flicker" on this spinner/message when it's being set on every for loop iteration?

i ask b/c if so, then maybe we only want to be calling `s.Suffix = ...` if/when `status.Message` changes (which would then force us to remove the `var err error` declaration and use something like `s, err := clients.Realm.DependenciesStatus(...)` so we can have reference to both past and current statuses

Reply:
I did not see any flicker when doing my testing so I don't think it's a problem. Unrelated to this, but I did notice a case of `" Installing dependencies: ...` during testing but this is because of the mock functions not providing any status messages during some tests. I could add status messages to the mock functions if this matters for tests.

Reply:
Awesome, and I wouldn't both with crafting messages for the tests, that text doesn't persist anyways (we don't assert on it)

I'd say all of this is good to go then!